reflexible/data_structures.py

```diff
@@ -565,3 +565,284 @@ def max(self):
     @property
     def min(self):
         return self._min
+
+
+class Command(object):
+    """ General COMMAND input for Flexpart 
+
+    #TODO: use properties ??
+    """
+
+
+    def __init__(self, **options):
+
+        self._OPTIONS = {
+
+            'IBDATE': [None , '''String simulation date start'''],
+            'IBDATE': [None , '''string, simulation time start'''],
+            'IEDATE': [None , '''string, simulation date end'''],
+            'IETIME': [None , '''string, simulation time end'''],
+            'LDIRECT': [1, '''Simulation direction, 1 for forward, -1 for backward in time'''],
+            'LOUTSTEP': [10800, '''Average concentrations are calculated every SSSSS seconds.'''],
+            'LOUTAVER': [10800, '''The average concentrations are time averages of SSSSS seconds
+                duration. If SSSSS is 0, instantaneous concentrations are outputted.'''],
```

Comment:
The above large string will include newlines and spaces.  The way to write large text messages for output is:

```
('The average concentrations are time averages of SSSSS seconds '  # note the trailing space
 'duration. If SSSSS is 0, instantaneous concentrations are outputted.')
```